authentik/flows/views/executor.py

```diff
@@ -171,7 +171,8 @@ def dispatch(self, request: HttpRequest, flow_slug: str) -> HttpResponse:
                     # Existing plan is deleted from session and instance
                     self.plan = None
                     self.cancel()
-                self._logger.debug("f(exec): Continuing existing plan")
+                else:
+                    self._logger.debug("f(exec): Continuing existing plan")
```

Comment:
This seems appropriate, but it changes logging behavior and I wanted to highlight that fact with this comment.